duckduckgo/Android
# Feature/default browser design feedback

Task/Issue URL: https://app.asana.com/0/35672304057861/608078252761679


**Description**:
Addressing feedback received while inside design review.

**Steps to test this PR**:
1. Test new onboarding asset used (variants `mx` and `mw` will show it) in 3rd onboarding screen
1. Verify _copy_ used on that screen ☝️ 
1. Test `atb` param includes variant when performing searches
1. Verify the dialog which popups upon clicking the notification banner (`mx` variant only) has its button with more radius on the corners



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
